components/Stack/Stack.tsx

```diff
@@ -15,7 +15,10 @@ const Wrapper = styled.div<Props>`
     props.shadow ? 'var(--baseBoxShadow)' : 'none'};
 
   border-radius: ${(props) =>
-    props.noRadius ? '0px' : ' var(--borderRadius)'};
```

Comment:
I'm a bit confused with the argument name. Is there a difference between `props` and `prop`? They seem to be used interchangeably.

Reply:
both are named `props` no?

Reply:
@YannCedric

Reply:
No difference, 
In an anonymous function, the argument object's name is irrelevant, the type is what matters. You can call it whatever you want, but it's still the same "thing"

Reply:
Thanks, it was a silly question, obs it's an anonym function, thanks!